README.md

```diff
@@ -42,14 +42,26 @@ The app has examples of basic interaction for both APS and ACS components.
 
 ## Yeoman generators
 
-To speed up the development of your ADF application, use the 
+To speed up the development of your ADF application, use the
 [Yeoman Generator](https://github.com/Alfresco/generator-ng2-alfresco-app).
 This will create a full working project with all the right libraries and tools.
 
 <p align="center">
   <img title="yeoman generator" src='https://github.com/yeoman/media/blob/master/optimized/yeoman-150x150-opaque.png' alt='yeoman logo'  />
 </p>
 
+## Activiti 7
```

Comment:
I guess this part of documentation is better to move in the process service cloud app